R/qenv-show.R

```diff
@@ -16,5 +16,26 @@
 #' @importFrom methods show
 #' @export
 setMethod("show", "qenv", function(object) {
-  rlang::env_print(object@.xData)
+  env <- get_env(object)
+  header <- cli::col_blue(sprintf("<environment: %s>", rlang::env_label(env)))
+  locked <- if (environmentIsLocked(env)) {
+    " [L]"
+  } else {
+    ""
+  }
+  parent <- sprintf("Parent: <environment: %s>", rlang::env_label(rlang::env_parent(env)))
+  cat(cli::style_bold(paste0(header, locked)), sep = "\n")
+  cat(parent, "\n")
+
+  shown <- ls(object)
```

Comment:
![image](https://github.com/user-attachments/assets/61c42894-063e-47be-874e-e61bb4bbdfed)
![image](https://github.com/user-attachments/assets/6a712d72-0489-4220-b099-002120403413)


```suggestion
  shown <- vapply(rlang::syms(ls(object)), deparse, character(1L), backtick = TRUE)
```

Reply:
Oh wait, this creates a problem with class after

Reply:
true